### PR TITLE
Framework/implement uart wrapper

### LINKED ISSUE #25: UART Wrapper
Implement a simple wrapper around the UART driver provided in the HAL

### PR DESCRIPTION
**Pull request by ChrisAmaya**

### CODE REVIEW

**Comment by Lars-Kish** [comment]
Instead of using bus2_uart, should the enum UARTbus be used in RUart.c?

File: radsat-sk/framework/wrappers/RUart.c [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+
+/**
+ * Receives data over UART and stores it in the given buffer
+ * @param data: A buffer to store the received data in
+ * @param size: The number of bytes to recieve over UART
+ * @return An integer error code. 0 indicates success
+ */
+uint32_t uartBlockingRecieve(uint8_t* data, uint16_t size) {
```
Comment (tslotboom) [inline radsat-sk/framework/wrappers/RUart.c]: There's a typo in this function, it's spelled receive

(i before e except after c or in weird words like weird)

File: radsat-sk/framework/wrappers/RUart.h [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
 #define RUART_H_
 
+#include <hal/Drivers/UART.h>
 
+#define CAMERA_BAUD_RATE 57600
+#define TIME_GUARD 0
+#define RX_TIMEOUT 255
+
+
+uint32_t UART_blockingTransmit(const uint8_t* data, uint16_t size);
```
Comment (tslotboom) [inline radsat-sk/framework/wrappers/RUart.h]: The names of these function declarations don't match the names of the function definitions in the c file

**Review by tslotboom — CHANGES_REQUESTED** [review]

**Comment by mpbtruong** [comment]
Looks good! Just pending those changes as per Thomas' comments.

**Review by mpbtruong — APPROVED** [review]

**Review by tslotboom — APPROVED** [review]

**Pull request merged**